pkg/catalog/catalog_test.go

```diff
@@ -113,28 +113,28 @@ func TestList(t *testing.T) {
 			})
 
 			// The function we are testing
-			output, err := List(client)
+			output, err := ListComponents(client)
 
 			//Checks for error in positive cases
 			if !tt.wantErr == (err != nil) {
-				t.Errorf("component List() unexpected error %v, wantErr %v", err, tt.wantErr)
+				t.Errorf("component ListComponents() unexpected error %v, wantErr %v", err, tt.wantErr)
 			}
 
 			// 1 call for current project + 1 call from openshift project for each of the ImageStream and ImageStreamTag
 			if len(fakeClientSet.ImageClientset.Actions()) != 4 {
-				t.Errorf("expected 2 ImageClientset.Actions() in List, got: %v", fakeClientSet.ImageClientset.Actions())
+				t.Errorf("expected 2 ImageClientset.Actions() in ListComponents, got: %v", fakeClientSet.ImageClientset.Actions())
 			}
 
 			// Check if the output is the same as what's expected (for all tags)
 			// and only if output is more than 0 (something is actually returned)
-			if len(output) > 0 && !(reflect.DeepEqual(output[0].AllTags, tt.wantAllTags)) {
-				t.Errorf("expected all tags: %s, got: %s", tt.wantAllTags, output[0].AllTags)
+			if len(output.Items) > 0 && !(reflect.DeepEqual(output.Items[0].Spec.AllTags, tt.wantAllTags)) {
```

Comment:
would be good to take the `Spec` out a variable to reduce verbosity and maybe a bit of cleanup?

Reply:
Yes I get that just saying `spec := output.Item[0].Spec` and use that at the two places . It's more of a nitpick.